vignettes/getting_started.Rmd

```diff
@@ -25,11 +25,14 @@ library(knitr)
 
 This package `glmtools` provides services in support using the General Lake Model (GLM) in R. GLM is a one-dimensional model that balances fluxes of energy and mass as part of dynamically simulating lake physics. The GLM model has been developed as an initiative of the Global Lake Ecological Observatory Network (GLEON) and in collaboration with the Aquatic Ecosystem Modelling Network (AEMON) that started in 2010. The model was first introduced in Leipzig at the 2nd Lake Ecosystem Modelling Symposium in 2012, and has since developed rapidly with application to numerous lakes within the GLEON network and beyond. Authors: Matthew R Hipsey, Louise C Bruce, David P Hamilton. More information about GLM can be found at <a href="http://aed.see.uwa.edu.au/research/models/GLM/">http://aed.see.uwa.edu.au/research/models/GLM/</a>.
 
+GLM version 3.0 was released in 2019. More information can be found in Hipsey et al 2019:
+<a hrfer="https://www.geosci-model-dev.net/12/473/2019/gmd-12-473-2019-discussion.html">https://www.geosci-model-dev.net/12/473/2019/gmd-12-473-2019-discussion.html.
+
 Running GLM requires three mandatory components (in addition to several optional components). The required components include:  
   
-1. A GLM executable for the operating system being used (currently there is support for linux, windows, and mavericks/yosemite mac OSX)  
-2. A `glm2.nml` file, which is a configuration file for the model (see below for specifics)  
-3. Appropriate driver data (currently in the form of a comma-delimited time-series file)
+1. A GLM executable for the operating system being used (currently there is support for linux, windows, and mojave mac OSX)  
+2. A `glm3.nml` file for version 3.0 or a `glm2.nml` file for version 2.2, which is a configuration file for the model (see below for specifics)  
```

Comment:
"file for versions 2.0-2.2"?

Reply:
Thanks Jordan. I'm going to resolve these comments as I incorporate them.